app/main/views/drafts.py

```diff
@@ -76,6 +100,78 @@ def copy_draft_service_from_existing_service(service_id):
     return single_result_response(RESOURCE_NAME, draft), 201
 
 
+@main.route('/draft-services/<framework_slug>/<lot_slug>/copy-published-from-framework', methods=['POST'])
+def copy_published_from_framework(framework_slug, lot_slug):
```

Comment:
A little docstring here might be nice to just expand on the name a little and give some more context so it's immediately obvious what this is doing.